TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift

```diff
@@ -55,10 +54,10 @@ public struct PushFactorPayload: FactorPayload {
     - serviceSid: The unique SID identifier of the Service.
     - identity: Identifies the user, should be an UUID you should not use PII (Personal Identifiable Information)
                 because the systems that will process this attribute assume it is not directly identifying information.
-    - pushToken: Registration token generated by APNS when registering for remote notifications.
+    - pushToken: (Optional) Registration token generated by the FCM SDK on initial startup of your app. Sending null or empty will disable sending push notifications for challenges associated to this factor.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/0c8d089e020078c52d469dcc5b1c3a5eb36c4ee1/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L57"></span>Line should be 195 characters or less: currently 207 characters<br />
<code>line_length</code> <code>FactorPayload.swift:57</code></td>
    </tr>
  </tbody>
</table>